lib/pages/rubnews/widgets/feed_item.dart

```diff
@@ -1,3 +1,4 @@
+import 'package:cached_network_image/cached_network_image.dart';
 import 'package:flutter/material.dart';
 import 'package:provider/provider.dart';
 import 'package:animations/animations.dart';
```

Comment:
Again here: In order to keep the order _dart, flutter, external packages, internal imports_ this should be moved down.
```suggestion
import 'package:flutter/material.dart';
import 'package:provider/provider.dart';
import 'package:animations/animations.dart';
import 'package:cached_network_image/cached_network_image.dart';
```